README.md

```diff
@@ -31,13 +31,25 @@ In this repository, you'll find a collection of [Jupyter notebooks](http://jupyt
 * [Work with the Usable Data Mask (UDM2)](https://github.com/planetlabs/notebooks/blob/master/jupyter-notebooks/udm2)
 
 ### Analyze and visualize Planet data
-* [Detect, count, and visualize ships in Planet imagery](https://github.com/planetlabs/notebooks/blob/master/jupyter-notebooks/ship-detector/01_ship_detector.ipynb)
-* [Python examples for remote sensing](https://github.com/kscottz/PythonFromSpace)
-* [Pixel-by-pixel comparison of PlanetScope and Landsat Scenes](https://github.com/planetlabs/notebooks/blob/master/jupyter-notebooks/landsat-ps-comparison/landsat-ps-comparison.ipynb)
-* [Calculate Coverage for a Search Query](https://github.com/planetlabs/notebooks/blob/master/jupyter-notebooks/coverage/calculate_coverage.ipynb)
-* [Segment and Classify Crops](https://github.com/planetlabs/notebooks/tree/master/jupyter-notebooks/crop-classification)
-* [Identify Forest Degradation](https://github.com/planetlabs/notebooks/tree/master/jupyter-notebooks/forest-monitoring)
-* [Identify the Temporal Signature of Crops](https://github.com/planetlabs/notebooks/tree/master/jupyter-notebooks/temporal-analysis)
+
+* Analytics quickstart:
+    1. [Summarizing Feeds and Subscriptions](notebooks/jupyter-notebooks/analytics/quickstart/01_checking_available_feeds_and_subscriptions.ipynb)
+    1. [Getting Analytic Feed Results](https://github.com/planetlabs/notebooks/blob/master/jupyter-notebooks/analytics/quickstart/02_fetching_feed_results.ipynb)
+    1. [Visualizing Raster Results](https://github.com/planetlabs/notebooks/blob/master/jupyter-notebooks/analytics/quickstart/03_visualizing_raster_results.ipynb)
+* Analytics user guide:
+    1. [Getting Started with Planet Analytics API](https://github.com/planetlabs/notebooks/blob/master/jupyter-notebooks/analytics/user-guide/1.%20Getting%20Started%20with%20the%20Planet%20Analytics%20API.ipynb)
+    1. [Planet Analytic Feeds Results](https://github.com/planetlabs/notebooks/blob/master/jupyter-notebooks/analytics/user-guide/2.%20Analytic%20Feeds%20Results.ipynb)
+* Other analytics notebooks:
+    * [Detect, count, and visualize ships in Planet imagery](https://github.com/planetlabs/notebooks/blob/master/jupyter-notebooks/ship-detector/01_ship_detector.ipynb)
+    * [Getting Started with the Planet Analytics API](https://github.com/planetlabs/notebooks/tree/master/jupyter-notebooks/analytics/1.%20Getting%20Started%20with%20the%20Planet%20Analytics%20API.ipynb)
```

Comment:
Duplicate link title from L40 above (and also invalid URL in this link) - remove

Reply:
removed